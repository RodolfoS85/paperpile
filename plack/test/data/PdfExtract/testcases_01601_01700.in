---
file: ../../../PDFREPOSITORY/001601.pdf
---
file: ../../../PDFREPOSITORY/001602.pdf
---
file: ../../../PDFREPOSITORY/001603.pdf
---
file: ../../../PDFREPOSITORY/001604.pdf
---
file: ../../../PDFREPOSITORY/001605.pdf
---
file: ../../../PDFREPOSITORY/001606.pdf
---
file: ../../../PDFREPOSITORY/001607.pdf
---
file: ../../../PDFREPOSITORY/001608.pdf
---
file: ../../../PDFREPOSITORY/001609.pdf
---
file: ../../../PDFREPOSITORY/001610.pdf
---
file: ../../../PDFREPOSITORY/001611.pdf
---
file: ../../../PDFREPOSITORY/001612.pdf
---
file: ../../../PDFREPOSITORY/001613.pdf
---
file: ../../../PDFREPOSITORY/001614.pdf
---
file: ../../../PDFREPOSITORY/001615.pdf
---
file: ../../../PDFREPOSITORY/001616.pdf
---
file: ../../../PDFREPOSITORY/001617.pdf
---
file: ../../../PDFREPOSITORY/001618.pdf
---
file: ../../../PDFREPOSITORY/001619.pdf
---
file: ../../../PDFREPOSITORY/001620.pdf
---
file: ../../../PDFREPOSITORY/001621.pdf
---
file: ../../../PDFREPOSITORY/001622.pdf
---
file: ../../../PDFREPOSITORY/001623.pdf
---
file: ../../../PDFREPOSITORY/001624.pdf
---
file: ../../../PDFREPOSITORY/001625.pdf
---
file: ../../../PDFREPOSITORY/001626.pdf
---
file: ../../../PDFREPOSITORY/001627.pdf
---
file: ../../../PDFREPOSITORY/001628.pdf
---
file: ../../../PDFREPOSITORY/001629.pdf
---
file: ../../../PDFREPOSITORY/001630.pdf
---
file: ../../../PDFREPOSITORY/001631.pdf
---
file: ../../../PDFREPOSITORY/001632.pdf
---
file: ../../../PDFREPOSITORY/001633.pdf
---
file: ../../../PDFREPOSITORY/001634.pdf
---
file: ../../../PDFREPOSITORY/001635.pdf
---
file: ../../../PDFREPOSITORY/001636.pdf
---
file: ../../../PDFREPOSITORY/001637.pdf
---
file: ../../../PDFREPOSITORY/001638.pdf
---
file: ../../../PDFREPOSITORY/001639.pdf
---
file: ../../../PDFREPOSITORY/001640.pdf
---
file: ../../../PDFREPOSITORY/001641.pdf
---
file: ../../../PDFREPOSITORY/001642.pdf
---
file: ../../../PDFREPOSITORY/001643.pdf
---
file: ../../../PDFREPOSITORY/001644.pdf
---
file: ../../../PDFREPOSITORY/001645.pdf
---
file: ../../../PDFREPOSITORY/001646.pdf
---
file: ../../../PDFREPOSITORY/001647.pdf
---
file: ../../../PDFREPOSITORY/001648.pdf
---
file: ../../../PDFREPOSITORY/001649.pdf
---
file: ../../../PDFREPOSITORY/001650.pdf
---
file: ../../../PDFREPOSITORY/001651.pdf
---
file: ../../../PDFREPOSITORY/001652.pdf
---
file: ../../../PDFREPOSITORY/001653.pdf
---
file: ../../../PDFREPOSITORY/001654.pdf
---
file: ../../../PDFREPOSITORY/001655.pdf
---
file: ../../../PDFREPOSITORY/001656.pdf
---
file: ../../../PDFREPOSITORY/001657.pdf
---
file: ../../../PDFREPOSITORY/001658.pdf
---
file: ../../../PDFREPOSITORY/001659.pdf
---
file: ../../../PDFREPOSITORY/001660.pdf
---
file: ../../../PDFREPOSITORY/001661.pdf
---
file: ../../../PDFREPOSITORY/001662.pdf
---
file: ../../../PDFREPOSITORY/001663.pdf
---
file: ../../../PDFREPOSITORY/001664.pdf
---
file: ../../../PDFREPOSITORY/001665.pdf
---
file: ../../../PDFREPOSITORY/001666.pdf
---
file: ../../../PDFREPOSITORY/001667.pdf
---
file: ../../../PDFREPOSITORY/001668.pdf
---
file: ../../../PDFREPOSITORY/001669.pdf
---
file: ../../../PDFREPOSITORY/001670.pdf
---
file: ../../../PDFREPOSITORY/001671.pdf
---
file: ../../../PDFREPOSITORY/001672.pdf
---
file: ../../../PDFREPOSITORY/001673.pdf
---
file: ../../../PDFREPOSITORY/001674.pdf
---
file: ../../../PDFREPOSITORY/001675.pdf
---
file: ../../../PDFREPOSITORY/001676.pdf
---
file: ../../../PDFREPOSITORY/001677.pdf
---
file: ../../../PDFREPOSITORY/001678.pdf
---
file: ../../../PDFREPOSITORY/001679.pdf
---
file: ../../../PDFREPOSITORY/001680.pdf
---
file: ../../../PDFREPOSITORY/001681.pdf
---
file: ../../../PDFREPOSITORY/001682.pdf
---
file: ../../../PDFREPOSITORY/001683.pdf
---
file: ../../../PDFREPOSITORY/001684.pdf
---
file: ../../../PDFREPOSITORY/001685.pdf
---
file: ../../../PDFREPOSITORY/001686.pdf
---
file: ../../../PDFREPOSITORY/001687.pdf
---
file: ../../../PDFREPOSITORY/001688.pdf
---
file: ../../../PDFREPOSITORY/001689.pdf
---
file: ../../../PDFREPOSITORY/001690.pdf
---
file: ../../../PDFREPOSITORY/001691.pdf
---
file: ../../../PDFREPOSITORY/001692.pdf
---
file: ../../../PDFREPOSITORY/001693.pdf
---
file: ../../../PDFREPOSITORY/001694.pdf
---
file: ../../../PDFREPOSITORY/001695.pdf
---
file: ../../../PDFREPOSITORY/001696.pdf
---
file: ../../../PDFREPOSITORY/001697.pdf
---
file: ../../../PDFREPOSITORY/001698.pdf
---
file: ../../../PDFREPOSITORY/001699.pdf
---
file: ../../../PDFREPOSITORY/001700.pdf
