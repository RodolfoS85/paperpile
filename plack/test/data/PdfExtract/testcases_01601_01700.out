---
authors: 'Sitch, S. and Cox, P. M. and Collins, W. J. and Huntingford, C.'
doi: 10.1038/nature06059
title: Indirect radiative forcing of climate change through ozone effects on the land-carbon sink
---
authors: 'Zhao, Xudong and Heng, Julian Ik-Tsen and Guardavaccaro, Daniele and Jiang, Richeng and Pagano, Michele and Guillemot, Francois and Iavarone, Antonio and Lasorella, Anna'
doi: 10.1038/ncb1727
title: The HECT-domain ubiquitin ligase Huwe1 controls neural differentiation and proliferation by destabilizing the N-Myc oncoprotein
---
authors: 'Hu, Yiguo and Liu, Yuhua and Pelletier, Shawn and Buchdunger, Elisabeth and Warmuth, Markus and Fabbro, Doriano and Hallek, Michael and Van Etten, Richard A and Li, Shaoguang'
doi: 10.1038/ng1343
title: 'Requirement of Src kinases Lyn, Hck and Fgr for BCR-ABL1-induced B-lymphoblastic leukemia but not chronic myeloid leukemia'
---
authors: 'Nissim, Lior and Beatus, Tsevi and Bar-Ziv, Roy'
doi: 10.1088/1478-3975/4/3/002
title: An autonomous system for identifying and governing a cell's state in yeast
---
authors: 'Li, Je-Luen and Car, Roberto and Tang, Chao and Wingreen, Ned S.'
doi: 10.1073/pnas.0610945104
title: Hydrophobic interaction and hydrogen-bond network for a methane pair in liquid water
---
authors: 'DE BOER, Paulo and Vos, Harmjan R. and Faber, Alex W. and Vos, Jan C. and Raue, Hendrik A.'
doi: 10.1261/rna.2257606
title: 'Rrp5p, a trans-acting factor in yeast ribosome biogenesis, is an RNA-binding protein with a pronounced preference for U-rich sequences'
---
authors: 'Fritz, Georg and Buchler, Nicolas E. and Hwa, Terence and Gerland, Ulrich'
doi: 10.1007/s11693-007-9006-8
title: 'Designing sequential transcription logic: a simple genetic circuit for conditional memory'
---
authors: 'De Raedt, Luc and Kersting, Kristian and Torge, Sunna'
title: Towards Learning Stochastic Logic Programs from Proof-Banks
---
authors: 'Reidys, Christian M.'
doi: 10.1007/s11538-008-9356-8
title: Local Connectivity of Neutral Networks
---
authors: 'Repoila, F. and Majdalani, N. and Gottesman, S.'
title: 'Small non-coding RNAs, co-ordinators of adaptation processes in Escherichia coli: the RpoS paradigm'
---
authors: 'RODOŠEK, Robert'
title: A Constraint-Based Approach for Deriving 3-D Structures of Cyclic Polypeptides
---
authors: 'Rombauts, Stephane and Florquin, Kobe and Lescot, Magali and Marchal, Kathleen and Rouze, Pierre and Van de Peer, Yves'
doi: 10.1104/pp.102.017715
title: 'Computational Approaches to Identify Promoters and cis- Regulatory Elements in Plant Genomes'
---
authors: 'Sachenbacher, Martin and Williams, Brian'
title: Diagnosis as Semiring-based Constraint Optimization
---
authors: 'Sandman, Karen E. and Noren, Christopher J.'
title: The efficiency of Escherichia coli selenocysteine insertion is influenced by the immediate downstream nucleotide
---
authors: 'Santalucia, Jr., John'
title: 'A unified view of polymer, dumbbell, and oligonucleotide DNA nearest-neighbor thermodynamics'
---
authors: 'Savill, Nicholas J. and Hoyle, David C. and Higgs, Paul G.'
title: 'RNA Sequence Evolution With Secondary Structure Constraints: Comparison of Substitution Rate Models Using Maximum-Likelihood Methods'
---
authors: 'Scherf, Matthias and Klingenhoff, Andreas and Werner, Thomas'
doi: 10.1006/jmbi.2000.3589
title: 'Highly Specific Localization of Promoter Regions in Large Genomic Sequences by PromoterInspector: A Novel Context Analysis Approach'
---
authors: 'Shih, C. T. and Su, Z. Y. and Gwan, J. F. and Hao, B. L. and Hsieh, C. H. and Lee, H. C.'
title: 'Mean-Field HP Model, Designability and Alpha-Helices in Protein Structures'
---
authors: 'Shortle, David and Chan, Hue Sun and Dill, Ken A.'
title: Modeling the effects of mutations on the denatured states of proteins
---
authors: 'Smith, Barbara and Stergiou, Kostas and Walsh, Toby'
title: Using auxiliary variables and implied constraints to model non-binary problems
---
authors: 'Tomita, Masaru and Hashimoto, Kenta and Takahashi, Kouichi and Shimizu, Thomas Simon and Matsuzaki, Yuri and Miyoshi, Fumihiko and Saito, Kanako and Tanida, Sakura and Yugi, Katsuyuki and Venter, J. Craig and Hutchison III, Clyde A.'
title: 'E-CELL: software environment for whole-cell simulation'
---
authors: 'Tomita, Masaru'
title: 'Whole-cell simulation: a grand challenge of the 21st century'
---
authors: 'Usuka, Jonathan and Zhu, Wei and Brendel, Volker'
title: Optimal spliced alignment of homologous cDNA to a genomic DNA template
---
authors: 'Vetcher, Alexandre A. and Napierala, Marek and Iyer, Ravi R. and Chastain, Paul D. and Griffith, Jack D. and Wells, Robert D.'
doi: 10.1074/jbc.M205209200
title: 'Sticky DNA, a Long GAA⅐GAA⅐TTC Triplex That Is Formed Intramolecularly, in the Sequence of Intron 1 of the Frataxin Gene'
---
authors: 'Zheng, C. L. and Nair, T. M. and Gribskov, M.'
title: A DATABASE DESIGNED TO COMPUTATIONALLY AID AN EXPERIMENTAL APPROACH TO ALTERNATIVE SPLICING
---
authors: 'Alber, Jochen and Gramm, Jens and Guo, Jiong and Niedermeier, Rolf'
title: Towards Optimally Solving the Longest Common Subsequence Problem for Sequences with Nested Arc Annotations in Linear Time
---
authors: 'Alkan, Can and Karakoc, Emre and Nadeau, Joseph H. and Sahinalp, S. Cenk and Zhang, Kaizhong'
title: 'RNA - RNA Interaction Prediction and Antisense RNA Target Search'
---
authors: 'Allali, Julien and Sagot, Marie-France'
title: A New Distance for High Level RNA Secondary Structure Comparison
---
authors: 'Beggs, Jean D. and Tollervey, David'
title: 'Crosstalk between RNA metabolic pathways: an RNOMICS approach'
---
authors: 'Blin, Guillaume and Touzet, Hlne'
title: 'How to Compare Arc-Annotated Sequences: The Alignment Hierarchy'
---
authors: 'Braich, Ravinderjit S. and Chelyapov, Nickolas and Johnson, Cliff and Rothemund, Paul W. K. and Adleman, Leonard'
title: Solution of a 20-Variable 3-SAT Problem on a DNA Computer
---
authors: 'Brenner, S Teven E. and Chothia, Cyrus and Hubbard, Tim J. P.'
title: Assessing sequence comparison methods with reliable structurally identified distant evolutionary relationships
---
authors: 'Busch, Anke and Will, Sebastian and Backofen, Rolf'
title: 'SECISDesign: A Server to Design SECIS- Elements within the Coding Sequence'
---
authors: 'Bystroff, Christopher and Thorsson, Vesteinn and Baker, David'
doi: 10.1006/jmbi.2000.3837
title: 'HMMSTR: a Hidden Markov Model for Local Sequence-Structure Correlations in Proteins'
---
authors: 'Claverie, Jean-Michel'
title: Computational methods for the identification of genes in vertebrate genomic sequences
---
authors: 'Davydov, Eugene and Batzoglou, Serafim'
title: A Computational Model for RNA Multiple Structural Alignment
---
authors: 'Demaine, Erik D. and Mozes, Shay and Rossman, Benjamin and Weimann, Oren'
title: An Optimal Decomposition Algorithm for Tree Edit Distance
---
authors: 'De Moor, Bart and Marchal, Kathleen and Mathys, Janick and Moreau, Yves and ESAT-SCD, '
title: 'Bioinformatics: Organisms from Venus, Technology from Jupiter, Algorithms from Mars'
---
authors: 'Dickson, Clive and Spencer-Dene, Bradley and Dillon, Christian and Fantl, Vera'
title: Tyrosine kinase signalling in breast cancer Fibroblast growth factors and their receptors
---
authors: 'Dovier, Agostino and Burato, Matteo and Fogolari, Federico'
title: Using Secondary Structure Information for Protein Folding in CLP (FD)
---
authors: 'Drapal, Nikola and Bock, August'
title: 'Interaction of the Hydrogenase Accessory Protein HypC with HycE, the Large Subunit of Escherichia coli Hydrogenase 3 during Enzyme Maturation'
---
authors: 'Flamm, Christoph and Hofacker, Ivo L. and Stadler, Peter F. and Wolfinger, Michael T.'
title: Barrier Trees of Degenerate Landscapes
---
authors: 'Flamm, Christoph and Ullrich, Alexander and Ekker, Heinz and Mann, Martin and Hgerl, Daniel and Rohrschneider, Markus and Sauer, Sebastian and Scheuermann, Gerik and Klemm, Konstantin and Hofacker, Ivo L. and Stadler, Peter F.'
title: Evolution of Metabolic Networks A Computational Framework
---
authors: 'Galgano, Alessia and Forrer, Michael and Jaskiewicz, Lukasz and Kanitz, Alexander and Zavolan, Mihaela and Gerber, Andre P.'
doi: 10.1371/journal.pone.0003164
title: 'Comparative Analysis of mRNA Targets for Human PUF- Family Proteins Suggests Extensive Interaction with the miRNA Regulatory System'
---
authors: 'Gerstein, Mark and Levitt, Michael'
title: 'Comprehensive assessment of automatic structural alignment against a manual standard, the scop classification of proteins'
---
authors: 'Graca, Ana and Lynce, Ins and Marques-Silva, Joo and Oliveira, Arlindo L.'
title: Haplotype Inference Combining Pedigrees and Unrelated Individuals
---
authors: 'Gupta, Mayetri and Liu, Jun S.'
doi: 10.1198/016214503388619094
title: Discovery of Conserved Sequence Patterns Using a Stochastic Dictionary Model
---
authors: 'Hendrix, Donna K. and Brenner, Steven E. and Holbrook, Stephen R.'
doi: 10.1017/S0033583506004215
title: 'RNA structural motifs : building blocks of a modular biomolecule'
---
authors: 'VAN HENTENRYCK, Pascal'
title: Search and Strategies in OPL
---
authors: 'Higgs, Paul G.'
title: 'RNA secondary structure : physical and computational aspects'
---
authors: 'Jansen, Ronald and Gerstein, Mark'
title: 'Analysis of the yeast transcriptome with structural and functional categories: characterizing highly expressed proteins'
---
authors: 'Kel, Alexander and Reymann, Susanne and Matys, Volker and Nettesheim, Paul and Wingender, Edgar and Borlak, Jürgen'
doi: 10.1124/mol.104.001677
title: 'A Novel Computational Approach for the Prediction of Networked Transcription Factors of Aryl Hydrocarbon- Receptor-Regulated Genes'
---
authors: 'Koschke, Rainer and Eisenbarth, Thomas'
title: A Framework for Experimental Evaluation of Clustering Techniques
---
authors: 'Ladd, Andrea N and Cooper, Thomas A'
title: 'Finding signals that regulate alternative splicing in the post- genomic era'
---
authors: 'Mann, Martin and Will, Sebastian and Backofen, Rolf'
title: 'CPSP-tools - Exact and Complete Algorithms for High- throughput 3D Lattice Protein Studies'
---
authors: 'Marcotte, Edward M. and Pellegrini, Matteo and Thompson, Michael J. and Yeates, Todd O. and Eisenberg, David'
title: 'A combined algorithm for genome- wide prediction of protein function'
---
authors: 'Meisner, Nicole-Claudia and Hackermüller, Jçrg and Uhl, Volker and Aszódi, Andrµs and Jaritz, Markus and Auer, Manfred'
doi: 10.1002/cbic.200400219
title: 'mRNA Openers and Closers: Modulating AU- Rich Element-Controlled mRNA Stability by a Molecular Switch in mRNA Secondary Structure'
---
authors: 'Michaels, GEORGES. and Carr, Daniel B.'
title: 'CLUSTER ANALYSIS AND DATA VISUALIZATION OF LARGE- SCALE GENE EXPRESSION DATA'
---
authors: 'Notredame, Cedric and Higgins, Desmond G. and Heringa, Jaap'
doi: 10.1006/jmbi.2000.4042
title: 'T-Coffee: A Novel Method for Fast and Accurate Multiple Sequence Alignment'
---
authors: 'Notredame, Cédric and Holm, Liisa and Higgins, Desmond G.'
title: 'COFFEE: an objective function for multiple sequence alignments'
---
authors: 'Oates, Tim and Doshi, Shailesh and Huang, Fang'
title: Estimating Maximum Likelihood Parameters for Stochastic Context-Free Graph Grammars
---
authors: 'Orlando, Valerio'
title: 'Mapping chromosomal proteins in vivo by formaldehyde- crosslinked-chromatin immunoprecipitation'
---
authors: 'Page, Roderic D. M.'
title: Comparative analysis of secondary structure of insect mitochondrial small subunit ribosomal RNA using maximum weighted matching
---
authors: 'Pal, Alessandro Dal and Dovier, Agostino and Pontelli, Enrico'
title: A New Constraint Solver for 3D Lattices and its Application to the Protein Folding Problem
---
authors: 'Pudimat, Rainer and Schukat-Talamazzini, Ernst-Günther and Backofen, Rolf'
title: Feature Based Representation and Detection of Transcription Factor Binding Sites
---
authors: 'DAlfonso, Giulio and Tramontano, Anna and Lahm, Armin'
doi: 10.1006/jsbi.2000.4351
title: 'Structural Conservation in Single-Domain Proteins: Implications for Homology Modeling'
---
authors: 'Biswas, Esther E. and Chen, Pei Hua and Biswas, Subhasis B.'
doi: 10.1021/bi00212a003
title: 'DNA helicase associated with DNA polymerase .alpha.: Isolation by a modified immunoaffinity chromatography'
---
authors: 'Erlich, Shlomit and Mizrachy, Liat and Segev, Oshik and Lindenboim, Liora and Zmira, Ofir and Adi-Harel, Sheli and Hirsch, Joel A. and Stein, Reuven and Pinkas-Kramarski, Ronit'
title: Differential Interactions Between Beclin 1 and Bcl-2 Family Members
---
authors: 'Furuya, Norihiko and Yu, Jie and Byfield, Maya and Pattingre, Sophie and Levine, Beth'
title: 'The Evolutionarily Conserved Domain of Beclin 1 is Required for Vps34 Binding, Autophagy and Tumor Suppressor Function'
---
authors: 'Casselton, Lorna A. and Olesnicky, Natalie S.'
title: Molecular Genetics of Mating Recognition in Basidiomycete Fungi
---
authors: 'Ilouze, Maya and Dishon, Arnon and Kotler, Moshe'
doi: 10.1128/MMBR.70.1.147-156.2006
title: Characterization of a Novel Virus Causing a Lethal Disease in Carp and Koi
---
authors: 'Anwar, Tamanna and Khan, Asad U'
title: Mapping and analysis of Simple Sequence Repeats in the Arabidopsis thaliana Genome
---
authors: 'Tanaka, Shigeo M. and Sun, Hui B. and Yokota, Hiroki'
title: Bone Formation Induced by a Novel Form of Mechanical Loading on Joint Tissue
---
authors: 'Fejtek, Monika and Delorme, Michel and Wassersug, Richard'
title: Behavioral Reactions of the Bat Carollia perspicillata to Abrupt Changes in Gravity
---
authors: 'Smolen, Paul and Baxter, Douglas A. and Byrne, John H.'
title: A Reduced Model Clarifies the Role of Feedback Loops and Time Delays in the Drosophila Circadian Oscillator
---
authors: 'Sharma, Shantanu and Ding, Feng and Dokholyan, Nikolay V.'
doi: 10.1529/biophysj.106.094805
title: Multiscale Modeling of Nucleosome Dynamics
---
authors: 'Coffman, Frederick D.'
title: Multiple Initiation Sites within the Human Ribosomal RNA Gene
---
authors: 'Rose, Nina F. and Marx, Preston A. and Luckay, Amara and Nixon, Douglas F. and Moretto, Walter J. and Donahoe, Sean M. and Montefiori, David and Roberts, Anjeanette and Buonocore, Linda and Rose, John K.'
title: An Effective AIDS Vaccine Based on Live Attenuated Vesicular Stomatitis Virus Recombinants
---
authors: 'Zhao, Weian and Chiuman, William and Brook, Michael A. and Li, Yingfu'
doi: 10.1002/cbic.200700014
title: Simple and Rapid Colorimetric Biosensors Based on DNA Aptamer and Noncrosslinking Gold Nanoparticle Aggregation
---
authors: 'Hara, Yasuhiro and Shimada, Kiyo and Kohga, Hiroshi and Ohara, Osamu and Koga, Hisashi'
title: 'High-throughput Production of Recombinant Antigens for Mouse KIAA Proteins in Escherichia coli: Computational Allocation of Possible Antigenic Regions, and Construction of Expression Plasmids of Glutathione-S-transferase-fused Antigens by an in vitro Recombination-assisted Method'
---
authors: 'Gatherer, Derek'
title: Peptide Vocabulary Analysis Reveals Ultra-Conservation and Homonymity in Protein Sequences
---
authors: 'Lin, Chun-Liang and Liu, Yuan-Wei and Chuang, Chia-Hua'
title: Control Design for Signal Transduction Networks
---
authors: 'Jayaswal, Vivek and Jermiin, Lars S. and Robinson, John'
title: Estimation of Phylogeny Using a General Markov Model
---
authors: 'Hook-Barnard, India G. and Hinton, Deborah M.'
title: 'Transcription Initiation by Mix and Match Elements: Flexibility for Polymerase Binding to Bacterial Promoters'
---
authors: 'Rosenfeld, Simon'
title: Patterns of Stochastic Behavior in Dynamically Unstable High-Dimensional Biochemical Networks
---
authors: 'Richard, Florence and Lombard, Martine and Dutrillaux, Bernard'
doi: 10.1101/gr.10.5.644
title: 'Phylogenetic Origin of Human Chromosomes 7, 16, and 19 and their Homologs in Placental Mammals'
---
authors: 'May, Thomas and Silverman, Ross D.'
title: 'Free-Riding, Fairness and the Rights of Minority Groups in Exemption from Mandatory Childhood Vaccination'
---
authors: 'Ganapathy, Ganeshkumar and Goodson, Barbara and Jansen, Robert and Le, Hai-son and Ramachandran, Vijaya and Warnow, Tandy'
title: Pattern Identification in Biogeography
---
authors: 'Lacroix, Vincent and Fernandes, Cristina G. and Sagot, Marie-France'
title: 'Motif Search in Graphs: Application to Metabolic Networks'
---
authors: 'Wiegel, Frederik W and Perelson, Alan S'
doi: 10.1046/j.0818-9641.2004.01229.x
title: Some Scaling Principles for the Immune System
---
authors: 'Movahedi, Sara and Waites, William'
title: A Two-Dimensional Protein Gel Electrophoresis Study of the Heat Stress Response of Bacillus subtilis Cells during Sporulation
---
authors: 'Loi, P. K. and Emmal, S. A. and Park, Y. and Tublitz, N. J.'
title: 'IDENTIFICATION, SEQUENCE AND EXPRESSION OF A CRUSTACEAN CARDIOACTIVE PEPTIDE (CCAP) GENE IN THE MOTH MANDUCA SEXTA'
---
authors: 'Payton, Mark E. and Greenstone, Matthew H. and Schenker, Nathaniel'
title: 'Overlapping confidence intervals or standard error intervals: What do they mean in terms of statistical significance?'
---
authors: 'Wang, Han'
doi: 10.1007/s00239-008-9121-5
title: Comparative Analysis of Period Genes in Teleost Fish Genomes
---
authors: 'Timmins, Graham S. and Jackson, Simon K. and Swartz, Harold M.'
doi: 10.1007/s002390010162
title: The Evolution of Bioluminescent Oxygen Consumption as an Ancient Oxygen Detoxification Mechanism
---
authors: 'Mirkin, Ekaterina V. and Mirkin, Sergei M.'
doi: 10.1128/MMBR.00030-06
title: Replication Fork Stalling at Natural Impediments
---
authors: 'Vieille, Claire and Zeikus, Gregory J.'
doi: 10.1128/MMBR.65.1.1-43.2001
title: 'Hyperthermophilic Enzymes: Sources, Uses, and Molecular Mechanisms for Thermostability'
---
authors: 'Ijuin, Takeshi and Yu, Y. Eugene and Mizutani, Kiyohito and Pao, Annie and Tateya, Sanshiro and Tamori, Yoshikazu and Bradley, Allan and Takenawa, Tadaomi'
doi: 10.1128/MCB.01990-06
title: Increased Insulin Action in SKIP Heterozygous Knockout Mice
---
authors: 'Copeland, Claudia S. and Matthews, Robert W. and GONZÁLEZ, Jorge M. and Aluja, Martin and Sivinski, John'
title: 'Wolbachia in Two Populations of Melittobia digitata Dahms (Hymenoptera: Eulophidae)'
---
authors: 'Halpin, David R. and Harbury, Pehr B.'
doi: 10.1371/journal.pbio.0020174
title: DNA Display II. Genetic Manipulation of Combinatorial Chemistry Libraries for Small-Molecule Evolution
